File: easy/1512/1512.js
/**
 * Runtime: 80 ms, faster than 47.12%.
 * Memory Usage: 38.4 MB, less than 56.81%.
 */
var numIdenticalPairs = function(nums) {
    let cnt = 0;
    for(let i=0; i<nums.length; ++i) {
        for(let j=i+1; j<nums.length; ++j) {
            if (nums[i] == nums[j]) ++cnt;
        }
    }
    return cnt;
};

File: easy/1512/1512.cpp
/**
* Runtime: 4 ms, faster than 54.71%
* Memory Usage: 7.5 MB, less than 56.34%
*/
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        int cnt = 0;
        for (int i=0; i<nums.size(); ++i)
        {
            for (int j=i+1; j<nums.size(); ++j)
            {
                if (nums[i] == nums[j]) ++cnt;
            }
        }
        
        return cnt;
    }
};
